metricbeat/module/ceph/_meta/output-examples/status.json

```diff
@@ -0,0 +1,101 @@
+{
+   "health":{
+      "health":{
+         "health_services":[
+            {
+               "mons":[
+                  {
+                     "name":"prcdsrvv1619",
+                     "kb_total":6281216,

```

Comment:
So far, I see that with REST API the output it's more simple. But I'll have a closer look at this.